keps/sig-network/4969-cluster-domain-downward-api/kep.yaml

```diff
@@ -0,0 +1,45 @@
+title: Cluster Domain Downward API
+kep-number: 4969
+authors:
+  - "@nightkr"
+owning-sig: sig-network
+participating-sigs:
+  - sig-network
+  - sig-node
+status: provisional
+creation-date: 2024-11-19
+reviewers:
+  - "@thockin"
+  - "@aojea"
+approvers:
+  - TBD
+
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.32"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: TBD
+  beta: TBD
+  stable: TBD
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: MyFeature
```

Comment:
We could pick the feature gate name even at provisional stage.

Reply:
Sure. `PodClusterDomain` would align with `PodHostIPs` (the only other downward API feature gate listed on https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/), but happy to hear other takes.